### PR TITLE
Add trademark note for Build Scan in documentation

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

**Comment by ljacomet** [comment]
@bot-gradle test this

File: subprojects/docs/src/docs/userguide/authoring-builds/multi-project/multi_project_builds.adoc [hunk 1]
```diff
@@ -157,5 +157,5 @@ This is already the de-facto pattern for many large projects.
 Besides, Gradle supports <<command_line_interface.adoc#sec:name_abbreviation,name abbreviation for kebab case names>>.
 
 3. _Define the root project name in the settings file_:
-The ´rootProject.name´ effectively assigns a name to the build as a whole, which is used in reports like build scans.
+The ´rootProject.name´ effectively assigns a name to the build as a whole, which is used in reports like a Build Scan™.
```
Comment (big-guy) [inline subprojects/docs/src/docs/userguide/authoring-builds/multi-project/multi_project_builds.adoc:160]:
```suggestion
The ´rootProject.name´ effectively assigns a name to the build as a whole, which is used in reports.
```

File: subprojects/docs/src/docs/userguide/build-cache/build_cache_debugging.adoc [hunk 2]
```diff
@@ -196,8 +196,8 @@ Build cache key for task ':compileJava' is 8ebf682168823f662b9be34d27afdf77
 The log shows e.g. which source files constitute the `stableSources` for the `compileJava` task.
 To find the actual differences between two builds you need to resort to matching up and comparing those hashes yourself.
 
-Luckily, you do not have to capture this data yourself - the build scan plugin already takes care of this for you.
-Gradle Enterprise has the necessary data to diagnose the cache miss when comparing two build scans, and will show you the properties that caused a different cache key to be generated.
+Luckily, you do not have to capture this data yourself - the Build Scan™ plugin already takes care of this for you.
+Gradle Enterprise has the necessary data to diagnose the cache miss when comparing two different Build Scan™, and will show you the properties that caused a different cache key to be generated.
```
Comment (big-guy) [inline subprojects/docs/src/docs/userguide/build-cache/build_cache_debugging.adoc:200]:
```suggestion
Gradle Enterprise has the necessary data to diagnose the cache miss when comparing two different Gradle builds, and will show you the properties that caused a different cache key to be generated.
```

File: subprojects/docs/src/docs/userguide/build-cache/build_cache_debugging.adoc [hunk 3]
```diff
@@ -206,13 +206,13 @@ There are three ways to compare builds in Gradle Enterprise.
...
 2. From a scan that you wish to compare with another, click the image:build-cache/compare-small.svg[compare,role="inline-icon",width="30"] icon at the top of the scan, then select the second scan by clicking the  image:build-cache/compare-small-b.svg[compare b,role="inline-icon",width="30"] icon from the list of scans.
-3. Click the image:build-cache/previous-next-scan.svg[workspace,role="inline-icon", width="22"] icon in the header of a build scan to show previous and subsequent build scans of the same workspace, then click the image:build-cache/compare-small-b.svg[compare b,role="inline-icon",width="30"] icon next to the build you want to compare.
+3. Click the image:build-cache/previous-next-scan.svg[workspace,role="inline-icon", width="22"] icon in the header of a Build Scan™ to show previous and subsequent build scans of the same workspace, then click the image:build-cache/compare-small-b.svg[compare b,role="inline-icon",width="30"] icon next to the build you want to compare.
```
Comment (big-guy) [inline subprojects/docs/src/docs/userguide/build-cache/build_cache_debugging.adoc:210]: Ugh, this section is full of references to just "scan" too and very outdated. (2) and (3) are completely different now.

Can we delete most of this and link to https://docs.gradle.com/enterprise/tutorials/task-inputs-comparison/?

File: subprojects/docs/src/docs/userguide/build-cache/build_cache_use_cases.adoc [hunk 4]
```diff
@@ -26,7 +26,7 @@ This helps to identify any possible issues with the build that may affect cachea
 
 If you are subject to audit requirements regarding the artifacts you ship to your customers you may need to disable the build cache for certain builds.
 Gradle Enterprise may help you with fulfilling these requirements while still using the build cache for all your builds.
-It allows you to easily find out which build produced an artifact coming from the build cache via build scans.
+It allows you to easily find out which build produced an artifact coming from the build cache via Build Scan™.
```
Comment (big-guy) [inline subprojects/docs/src/docs/userguide/build-cache/build_cache_use_cases.adoc:29]:
```suggestion
It allows you to easily find out which build produced an artifact coming from the build cache via the Build Scan™ service.
```

File: subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc [hunk 5]
```diff
@@ -4,20 +4,20 @@
 Gradle provides sufficient tooling to navigate large dependency graphs and mitigate situations that can lead to link:https://en.wikipedia.org/wiki/Dependency_hell[dependency hell].
 Users can choose to render the full graph of dependencies as well as identify the selection reason and origin for a dependency.
 The origin of a dependency can be a declared dependency in the build script or a transitive dependency in graph plus their corresponding configuration.
-Gradle offers both capabilities through visual representation via build scans and as command line tooling.
+Gradle offers both capabilities through visual representation via Build Scan™ and as command line tooling.
```
Comment (big-guy) [inline subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc:7]:
```suggestion
Gradle offers both capabilities through visual representation with command-line tooling and the Build Scan™ service.
```

File: subprojects/docs/src/docs/userguide/extending-gradle/task_configuration_avoidance.adoc [hunk 6]
```diff
@@ -67,8 +67,8 @@ We also cover some <<#sec:task_configuration_avoidance_troubleshooting, troubles
 === General
 1. [[task_configuration_avoidance_guideline_use_help_task]] **Use `help` task as a benchmark during the migration.**
 The `help` task is the perfect candidate to benchmark your migration process.
-In a build that uses only the configuration avoidance API, https://scans.gradle.com/s/o7qmlmmrsfxz4/performance/configuration?openScriptsAndPlugins=WzFd[a build scan would show no tasks created immediately or created during configuration], and only the tasks that were actually executed would be created.
-Be mindful of <<#task_configuration_avoidance_pitfall_build_scan_plugin, the version of the build scan plugin in use>>.
+In a build that uses only the configuration avoidance API, https://scans.gradle.com/s/o7qmlmmrsfxz4/performance/configuration?openScriptsAndPlugins=WzFd[a Build Scan™ would show no tasks created immediately or created during configuration], and only the tasks that were actually executed would be created.
+Be mindful of <<#task_configuration_avoidance_pitfall_build_scan_plugin, the version of the Build Scan™ plugin in use>>.
```
Comment (big-guy) [inline subprojects/docs/src/docs/userguide/extending-gradle/task_configuration_avoidance.adoc:71]: This is old advice. I don't think you can use a version of the plugin that this warning applies to and a new version of Gradle.
```suggestion
```

File: subprojects/docs/src/docs/userguide/extending-gradle/task_configuration_avoidance.adoc [hunk 7]
```diff
@@ -335,7 +336,7 @@ task("anEagerTask") {
 =====
 ====
 
-* [[task_configuration_avoidance_pitfall_build_scan_plugin]] **The build scan plugin `buildScanPublishPrevious` task is eager until version 1.15.** Upgrade the build scan plugin in your build to use the latest version.
+* [[task_configuration_avoidance_pitfall_build_scan_plugin]] **The Build Scan™ plugin `buildScanPublishPrevious` task is eager until version 1.15.** Upgrade the Build Scan™ plugin in your build to use the latest version.
```
Comment (big-guy) [inline subprojects/docs/src/docs/userguide/extending-gradle/task_configuration_avoidance.adoc:339]:
```suggestion
```

File: subprojects/docs/src/docs/userguide/licenses.adoc [hunk 8]
```diff
@@ -28,6 +28,6 @@ Gradle user manual and DSL references are licensed under link:http://creativecom
 
 
 [[licenses:build_scan_plugin]]
-== Gradle Build Scan Plugin
+== Gradle Build Scan™ Plugin
```
Comment (big-guy) [inline subprojects/docs/src/docs/userguide/licenses.adoc:31]:
```suggestion
== Gradle Enterprise Plugin
```

**Review by big-guy — COMMENTED** [review]
half way there, livin' on a prayer.

**Comment by ljacomet** [comment]
On hold, let's see if all these changes are really needed.

**Comment by ljacomet** [comment]
Closing as not needed

**Pull request closed without merging**